src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
```

Comment:
Let's check with the checklist

https://github.com/mate-academy/layout_html-form/blob/master/checklist.md

Reply:
> Let's check with the checklist
> 
> https://github.com/mate-academy/layout_html-form/blob/master/checklist.md

I have to put closing ">" on the next line to match the code style from the checklist, right?